src/main/scala/Query.scala

```diff
@@ -1,86 +0,0 @@
-package com.originate.scalypher
```

Comment:
i suspect it's the move + significantly changing content in the new Query.scala, git diff probably shows the large changes too